Exercise_A/index.html

```diff
@@ -1,45 +1,190 @@
 
-    <!-- 
-      This is where we declare all of the external libraries our page uses
-      
-      You will need:
-      Animate CSS
-      Bootstrap CSS
-      You own custom stylesheet
-      FontAwesome
+  <title>Solution for day 4 using Bootstrap v4</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css" />
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css"
+    integrity="undefined" crossorigin="anonymous">
+  <link rel="stylesheet" href="assets/css/style.css">
+  <script src="https://kit.fontawesome.com/78ccc250b3.js" crossorigin="anonymous"></script>
+</head>
 
-      Refer to the Practice directory
-    -->
-  </head>
+<!-- Nav bar setup -->
 
-  <!-- Nav bar setup -->
+<body data-spy="scroll" data-target=".navbar">
+  <nav class="navbar navbar-expand-lg navbar-dark bg-transparent justify-content-between fixed-top p-2 border-bottom border-light">
+    <a class="navbar-brand" href="#">
+      <div id="logo">THE ONLY BOOTSTRAP PAGE</div>
+    </a>
+    <ul class="navbar-nav">
+      <li class="nav-item">
+        <a class="nav-link" href="#about">About</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="#fouricons">Four Icons</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="#bullshit">Bullshit</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="#callme">Call Me</a>
+      </li>
+    </ul>
+  </nav>
+  <div class="jumbotron text-center d-flex align-items-center justify-content-center text-white p-2">
+    <div>
+      <h1 class="display-4 w-75 mx-auto font-weight-bold">HEY LOOK, IT'S EVERY BOOTSTRAP WEBSITE EVER</h1>
+      <hr>
+      <p class="lead w-75 mx-auto text-light p-2">Take a look around at the same fucking bootstrap page you've seen ten
+        million times before! Featuring all the same things as those pages!</p>
+      <p class="lead">
+        <a class="btn btn-danger btn-lg rounded-pill" href="#" role="button">Learn more</a>
+      </p>
+    </div>
+  </div>
+  <div id="about" class="row p-5 d-inline-flex justify-content-center">
+    <div class="text-center p-5">
+      <h2 class="text-light w-50 mx-auto p-2">Want to make an original website yourself?</h2>
+      <hr id="lighthr" class="mx-auto">
+      <p class="text-light  w-50 mx-auto p-2">Forget that! Who would ever want to put in all of that effort for a
+        website? Just open up
+        your web browser
+        and type "bootstrap template" into your favorite search engine, like Yahoo! or Bing, and you're on your way!
+        There are hundreds of templates to choose from, but go ahead and pick this same exact template from the first
+        result on google, edit a few lines, and you're on your way! No one will notice!</p>
+      <button type="button" class="btn btn-light rounded-pill">GOOGLE THAT SHIT</button>

```

Comment:
You should nest an a tag in here that send the user to the same URL of the original website